main/webapp/modules/core/styles/project/sidebar.css

```diff
@@ -251,3 +251,7 @@ textarea.history-operation-json {
   min-height: 450px;
   min-width: 374px;
 }
+
+.history-operation-json-error {
+  color: var(--error-red);
```

Comment:
The error currently renders in black because of a dependency of this PR on #6903.